src/kudu/consensus/leader_election.cc

```diff
@@ -565,23 +559,21 @@ FlexibleVoteCounter::IsMajoritySatisfiedInMajorityOfRegions() const {
       satisfaction_possible_count >= num_majority_regions);
 }
 
-std::pair<bool, bool>
-FlexibleVoteCounter::IsMajoritySatisfiedInPotentialLeaderRegions(
-    const std::set<std::string>& leader_regions) const {
-  CHECK(!leader_regions.empty());
-  std::vector<std::string> leader_regions_vector(
-      leader_regions.begin(), leader_regions.end());
+std::pair<bool, bool> FlexibleVoteCounter::IsMajoritySatisfiedInAllRegions(
```

Comment:
thinking about it a bit more, is "All" a bit confusing or should it be "These" ?